env.js

```diff
@@ -28,6 +28,7 @@ const defaults = {
   OC_APPLICATION: 'frontend',
   OC_ENV: process.env.NODE_ENV || 'development',
   OC_SECRET: crypto.randomBytes(16).toString('hex'),
+  WISE_ENVIRONMENT: process.env.NODE_ENV === 'production' ? 'production' : 'sandbox',
```

Comment:
I think we rather want to use `OC_ENV`, esp. to differentiate between prod and staging

```suggestion
  WISE_ENVIRONMENT: process.env.OC_ENV === 'production' ? 'production' : 'sandbox',
```